src/app/contextBroker/contextBroker.cpp

```diff
@@ -1239,20 +1243,26 @@ int main(int argC, char* argV[])
 
   if (https)
   {
-    // FIXME P3: we suspect that loadFile() is not working well as it doesn't include the
-    // char terminator \0 at the end, thus causing valgrind errors. Using calloc will ensure
-    // that all the buffer is initialized with \0, thus avoiding the problem, although it would
-    // be better to fix loadFile()
-    // See issue https://github.com/telefonicaid/fiware-orion/issues/3925 for more detail
```

Comment:
Review if valgrind pass is disabled in some .test due to this, re-enabling them again in that case.

Reply:
It should not be part of the content of this PR, If there is a previous process disabled due to the implementation of the loadFile()

Reply:
Related PR https://github.com/telefonicaid/fiware-orion/pull/3926 (where that FIXME note was included)

I think the only affected .test would be [test/functionalTest/cases/0000_https_support/https.test](https://github.com/telefonicaid/fiware-orion/blob/master/test/functionalTest/cases/0000_https_support/https.test), which already  have `VALGRIND_READY`.

Thus, NTC